linode_api4/objects/linode.py

```diff
@@ -387,6 +387,26 @@ def available_backups(self):
             }))
 
         return self._avail_backups
+    
+    def reset_instance_root_password(self, root_password=None):
+        rpass = root_password
+        if not rpass:
+            rpass = Instance.generate_root_password()
+
+        params = {
+            'password': rpass,
+        }
+        
+        self._client.post('{}/password'.format(Instance.api_endpoint), model=self, data=params)
+
+    
+    def transfer_year_month(self, year, month):
+        """
+        Upgrades this Instance to the latest generation type
+        """
+
+        self._client.get('{}/transfer/{}/{}'.format(Instance.api_endpoint, year, month), model=self)
```

Comment:
Could we convert the result here to a `MappedObject(...)` and return it. This allows users to access attributes using `.` syntax.

e.g. 
```
return MappedObject(**result)
```